src/app/[locale]/incident/add/components/FeatureListItem.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { Dispatch, RefObject, SetStateAction, useMemo } from 'react'
+import { getFeatureType } from '@/lib/utils/map'
+import { Feature, FeatureCollection } from 'geojson'
+import { PublicQuestion } from '@/types/form'
+import { MapRef } from 'react-map-gl/maplibre'
+import { FormField, FormFieldCheckbox, Icon } from '@/components/index'
+import { useTranslations } from 'next-intl'
+import { FeatureWithDescription } from '@/types/map'
+import { useFormStore } from '@/store/form_store'
+
+type FeatureListItemProps = {
+  feature: FeatureWithDescription
+  field: PublicQuestion
+  map: MapRef | undefined
+  setError: Dispatch<SetStateAction<string | null>>
+  dialogRef: RefObject<HTMLDialogElement>
+  configUrl?: string
+}
+
+export const FeatureListItem = ({
+  feature,
+  field,
+  map,
+  setError,
+  dialogRef,
+  configUrl,
+}: FeatureListItemProps) => {
+  const t = useTranslations('describe-add.map')
+  const { formState, updateForm } = useFormStore()
+
+  const featureId = feature.id
+  const featureDescription = feature.description
+  const maxNumberOfAssets = field ? field.meta.maxNumberOfAssets : 1
+
+  // Get feature type of asset
+  const featureType = useMemo(() => {
+    return getFeatureType(field.meta.featureTypes, feature.properties)
+  }, [field.meta.featureTypes, feature.properties])
+
+  // Add or remove feature to / from the newSelectedFeature state declared in DialogMap
+  const addOrRemoveFeature = (value: boolean) => {
```

Comment:
Hier zouden we misschien een wat meer beschrijvende naam voor kunnen gebruiken zoals checked oid